src/config.ts

```diff
@@ -27,6 +27,6 @@ export const networkConfigs = {
   },
   localhost: {
     label: 'Local Network',
-    url: 'http://localhost:3000/wallet/network-config',
+    url: 'https://wallet.agoric.app/wallet/network-config',
```

Comment:
why change this?  not a big deal but makes it so I can't test on a plane :)

Reply:
I was getting cors issues when testing from the cloudflare deployment, tradeoffs I guess...